META-zip

```diff
@@ -3,3 +3,5 @@ requires="unix"
 archive(byte)="zip.cma"
 archive(native)="zip.cmxa"
 archive(native,plugin)="zip.cmxs"
```

Comment:
I'm going to assume this line `archive(native,plugin)="zip.cmxs"` is still useful, or at least doesn't hurt.